physicsnemo/models/domino/model.py

```diff
@@ -333,15 +358,21 @@ class GeometryRep(nn.Module):
     Geometry representation module that processes STL geometry data.
 
     This module constructs a multiscale representation of geometry by:
-    1. Computing short-range geometry encoding for local features
-    2. Computing long-range geometry encoding for global context
-    3. Processing signed distance field (SDF) data for surface information
+    1. Computing multi-scale geometry encoding for local and global context
+    2. Processing signed distance field (SDF) data for surface information
 
     The combined encoding enables the model to reason about both local and global
     geometric properties.
     """
 
-    def __init__(self, input_features: int, radii, model_parameters=None):
+    def __init__(
+        self,
+        input_features: int,
+        radii,
```

Comment:
```suggestion
        radii: Sequence[float],
```
and `from typing import Sequence`